src/test/tests/databases/adios2.py

```diff
@@ -47,31 +53,31 @@ def set_test_view(tag_name):
     else:
         ResetView()
 
-def test(mesh_name, tag_name):
-    AddPlot("Mesh", mesh_name, 1, 1)
-    AddPlot("Pseudocolor", "/data/0/meshes/admbase_lapse_rl00/admbase_alp", 1, 1)
-    AddOperator("Clip", 1)
-    SetActivePlots(1)
-    SetActivePlots(1)
-    ClipAtts = ClipAttributes()
-    ClipAtts.quality = ClipAtts.Fast  # Fast, Accurate
-    ClipAtts.funcType = ClipAtts.Plane  # Plane, Sphere
-    ClipAtts.plane1Status = 1
-    ClipAtts.plane2Status = 0
-    ClipAtts.plane3Status = 0
-    ClipAtts.plane1Origin = (35, 35, 35)
-    ClipAtts.plane2Origin = (0, 0, 0)
-    ClipAtts.plane3Origin = (0, 0, 0)
-    ClipAtts.plane1Normal = (1, 0, 0)
-    ClipAtts.plane2Normal = (0, 1, 0)
-    ClipAtts.plane3Normal = (0, 0, 1)
-    ClipAtts.planeInverse = 0
-    ClipAtts.planeToolControlledClipPlane = ClipAtts.Plane1  # NONE, Plane1, Plane2, Plane3
-    ClipAtts.center = (0, 0, 0)
-    ClipAtts.radius = 1
-    ClipAtts.sphereInverse = 0
-    ClipAtts.crinkleClip = 0
-    SetOperatorOptions(ClipAtts, 0, 1)
+def test(mesh_name, tag_name, var_name):
+    # AddPlot("Mesh", mesh_name)
```

Comment:
Why remove the Mesh plot from being part of this test?

Reply:
The mesh plot produces a filled-in black square for the latter two tests, because of the density of the meshlines. It completely obscures the pseudocolor plot. I could add separate tests for mesh plots. What do you think?

Reply:
You could separate, or pass in a flag to the function specifying if you want a mesh plot (true for the original test, false for the bp tests).

Reply:
Ok, I fixed this.